pdns/recursordist/settings/table.py

```diff
@@ -1715,7 +1715,7 @@
         'section' : 'nod',
         'oldname' : 'new-domain-history-dir',
         'type' : LType.String,
-        'default' : '/usr/local/var/lib/pdns-recursor/nod',
+        'default' : 'NODCACHEDIRNOD',
```

Comment:
In my tests this ends up as `• Default: NODCACHEDIRNOD` in the documentation, which feels a bit weird. Is that intended?